printer_mods/jeoje/Z_Chain_Guide_Thermistor_Mount/README.md

```diff
@@ -0,0 +1,20 @@
+# Voron V2.4 Z Chain Guide with Integrated Thermistor Mount
+
+When looking for a clean way to mount a chamber temperature sensor, the Z chain guide seemed like a perfect opportunity.  It should fit any thermistor with a 3mm barrel.
+
+<div align="center">
+    <img src="images/render1.png" height="350" hspace="10">
+    <img src="images/render2.png" height="350">
+</div>
+
+
+## Installation
+1. Route your wire through the Z chain to the rear X extrusion
+2. Secure the wire inside that extrusion
+3. Insert the thermistor into the mount
+4. Attach the mount to the extrusion, positioned to hold the Z chain vertical*
+
+****NOTE:** Ensure you have enough links in your Z chain to clear the guide when at max Z.*
+
```

Comment:
It looks like part of the thermistor hole prints in air:

https://user-images.githubusercontent.com/69389593/112765421-1f429880-8fdb-11eb-872f-1f612b908aa5.mov

Reply:
Fixed in commit [793fcb1](https://github.com/VoronDesign/VoronUsers/pull/251/commits/793fcb1e8944c2bc7a76a521c5d505614edca49d)